README.md

```diff
@@ -286,6 +287,9 @@ If the `jest.mock()` calls is placed after actual code, (e.g. after functions or
 the line numbers in stacktraces will be off.
 We suggest placing the `jest.mock()` calls after the imports, but before any actual code.
 
+### Current limitations for debugger statements
+The `debugger`; statements do not work in VSCode, but breakpoints will map the TypeScript code correctly to the transpiled Javascript.
+ 
```

Comment:
Shouldn't this be the other way around - ''...will map the transpiled Javasript code correctly to the Typescript code"?

Reply:
I just reused the same documentation which was removed couple of days back https://github.com/kulshekhar/ts-jest/commit/9cb6f86d152773f2bddaf9234db99c89b5c4e568#diff-04c6e90faac2675aa89e2176d2eec7d8

It works both ways I guess?

Reply:
I quickly went through definitions of source maps online.
Generic definition suggests it's a mapping of compressed/transpiled code to source file as described [here](http://blog.teamtreehouse.com/introduction-source-maps) or [here](https://stackoverflow.com/a/17493808)  

I'll update the PR.